components/inputs/input-time.js

```diff
@@ -112,7 +113,8 @@ class InputTime extends LitElement {
 			 */
 			maxHeight: { type: Number, attribute: 'max-height' },
 			/**
-			 * Number of minutes between times shown in dropdown. Valid values are "five", "ten", "fifteen", "twenty", "thirty", "sixty".
+			 * Number of minutes between times shown in dropdown
+			 * @type {('five'|'ten'|'fifteen'|'twenty'|'thirty'|'sixty')}
```

Comment:
We don't need to call out the `@default` here because lit-analyzer gets it from the constructor, ya?

Reply:
Yup that's correct.